crates/typst/src/foundations/int.rs

```diff
@@ -145,7 +145,6 @@ impl i64 {
     #[func(title = "Bitwise Left Shift")]
     pub fn bit_lshift(
         self,
-
```

Comment:
I think this change (empty line deletion) can be undone. Also on line 171 and line 181 below. You can check them in git diff.

Reply:
I also found it weird at first, but I think other bitwise operation functions don't have this empty line, so this probably makes it more consistent.

Reply:
I will keep it then!